lib/src/rmui_bundle_suggestors/html_script_adder.dart

```diff
@@ -0,0 +1,62 @@
+// Copyright 2021 Workiva Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'package:codemod/codemod.dart';
+import 'package:over_react_codemod/src/rmui_bundle_suggestors/dart_script_adder.dart';
+
+import 'constants.dart';
+
+typedef GetScriptToAdd = String Function({String pathPrefix});
+
```

Comment:
My IDE is telling me this is unused. Was there are a reason to keep it around (or is my IDE wrong)?
```suggestion
```

Reply:
Good catch... for some reason it seems lints aren't showing up in my IDE 😕

Reply:
Ohh or maybe it's a non-Dart-analysis-server inspection that I have disabled?